apps/web/ee/components/eventtype/AttendeeReminderTypeForm.tsx

```diff
@@ -62,7 +61,7 @@ const CustomInputTypeForm: FC<Props> = (props) => {
           render={({ field }) => (
             <Select
               id="method"
-              defaultValue={selectedMethodOption}
+              defaultValue={selectedAttendeeReminder ? selectedMethodOption : methodOptions[0].value}
```

Comment:
Default reminder is SMS

Reply:
We should not show this options if TWILIO keys aren't set

Reply:
Updated some changes about this